### ISSUE TITLE
Ciphertext auto-clear not working

### ISSUE THREAD

**Issue by MatthewBieda** [pglombardo/PasswordPusher#654]
## 🐛 Bug Report

Hi we have a branded version successfully deployed as an Azure Web App connected to a managed MySql database. We are using Github Actions to build a new version on changes to the repo and push the image to Azure Container Registry which refreshes the Web App via a webhook. 

I just noticed an issue with the database failing to automatically wipe the ciphertext after the set expiry period. It remains present until you try to actually visit the link and it notifies the user it has expired. I have verified this only occurs with redeployments of the app, if we leave a long running instance the auto-expiry works fine. All other functionality of the app is fine.

I was wondering if you might be able to provide any insight here?

Thank you,
Matt

### Environment

- [X] Docker
  - [X] pwpush-mysql
- [X] Microsoft Azure

### Screenshots

![automatic_deletion](https://user-images.githubusercontent.com/63665300/211793458-edd6a8e3-433f-4a0b-a644-08e714c64e8c.jpg)

**Comment by pglombardo** [comment]
Hi @MatthewBieda - To preemptively and automatically expire payloads, the application would need a background job server and datastore (e.g. Redis) to periodically scan for pushes that have expired.

For simplicity (for myself and for users), I decided against packaging two additional required components with the application.  The default action is to validate that the push is not expired before serving it as you noted.

And to add to that strategy, also provided in the container are [tasks that can be scheduled](https://github.com/pglombardo/PasswordPusher/wiki/Periodic-Background-Tasks) to run (e.g. via cron) to preemptively expire pushes.  That page also has a more extensive explanation of reasoning and function in the [Notes section](https://github.com/pglombardo/PasswordPusher/wiki/Periodic-Background-Tasks#note).

tldr; I think what you are looking for are [these background tasks](https://github.com/pglombardo/PasswordPusher/wiki/Periodic-Background-Tasks) that you can run periodically to preemptively expire pushes.

Let me know if this answers your questions sufficiently...

**Comment by MatthewBieda** [comment]
Yes thanks for your response I will look into running those!
